### DISCUSSION TITLE
Update verification naming

### DISCUSSION THREAD

**Discussion by Ferrariic** [Bot-detector/Bot-Detector-Core-Files#447] [Ideas]
Current verification naming methods for permissions are not clear as to what permissions they actually give. 

It would be preferable to change the current naming convention to:

- Plugin
- ML
- Discord
- Twitter
- Staff
- Admin

This would help clarify what permissions are expected of what fields. These would be grouped into: Public, Internal & Admin groups.

**Comment by Ferrariic** [comment]
We may need an admin panel prior to making this change, so that new tokens can be created easily with the necessary perms.

**Comment by extreme4all** [comment]
these are the users:

- Plugin
- ML
- Discord
- Twitter
- Staff
- Admin

the groups could be

- public (plugin)
- administrative or staff (staff)
- internal (ML, discord, twitter)
- Admin
